config/pitest-suppressions/pitest-javadoc-suppressions.xml

```diff
@@ -522,6 +531,15 @@
     <lineContent>return builder.toString().trim();</lineContent>
   </mutation>
 
+  <mutation unstable="false">
+    <sourceFile>JavadocStyleCheck.java</sourceFile>
```

Comment:
@romani to be clear, you are good to extend list of surviving mutations?

Reply:
No extension is allowed for technical dept.
Especially for this questionable update, as we have better Check to validate it.

Reply:
@0xbakry, please update tests to cover this mutation survival.

Reply:
Sorry for delay, I was a little bit busy with my final exams, I'll try to update it soon.

Reply:
@0xbakry, ping, do we have chance to finish this PR

Reply:
I will close it for now, till I finish my GP and I'll be back.